README.md

```diff
@@ -1,2 +1,31 @@
 # Hello-Microverse
 This is a hello microverse project for my course in Microverse.
+In this project, you will set up a "Hello world" repository. 
+No complex coding is required for this exercise. 
```

Comment:
- [ ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

Reply:
Good job implementing the changes asked by the previous reviewer. 👍 
Your readme is perfect!